calour/filtering.py

```diff
@@ -458,7 +458,10 @@ def filter_samples(exp: Experiment, field, values, negate=False, inplace=False):
         the filtered object
 
     '''
-    values = _to_list(values)
+    # if it is None - pass to filter_by_metadata so will remove the NaN samples
```

Comment:
i think this behavior is not intuitive.

Reply:
We need a convenient way to filter samples with None in metadata (since read() also reads samples from the biom table not present in the mapping file and puts None.
We have this behavior for filter_by_metadata() we just pass the parameter.
i think it is good. any other suggestions?